rosdep/python.yaml

```
@@ -5965,6 +5965,7 @@ python3-filelock:
   arch: [python-filelock]
   debian: [python3-filelock]
   fedora: [python3-filelock]
+  nixos: [python3Packages.filelock]
```

The link you provide is for python313Packages.filelock
```suggestion
  nixos: [python313Packages.filelock]
```

Is this a valid alias or does it need to be more specific like this?